meta/src/main/java/org/apache/rocketmq/mqtt/meta/raft/processor/StateProcessor.java

```diff
@@ -16,16 +16,30 @@
  */
 
 package org.apache.rocketmq.mqtt.meta.raft.processor;
+
+import com.alipay.sofa.jraft.storage.snapshot.SnapshotReader;
+import com.alipay.sofa.jraft.storage.snapshot.SnapshotWriter;
 import org.apache.rocketmq.mqtt.common.model.consistency.ReadRequest;
 import org.apache.rocketmq.mqtt.common.model.consistency.Response;
 import org.apache.rocketmq.mqtt.common.model.consistency.WriteRequest;
+import org.apache.rocketmq.mqtt.meta.raft.snapshot.SnapshotOperation;
+
+import java.util.function.BiConsumer;
 
 public abstract class StateProcessor {
 
     public abstract Response onReadRequest(ReadRequest request);
 
     public abstract Response onWriteRequest(WriteRequest log);
 
+    public SnapshotOperation loadSnapshotOperate() {
```

Comment:
onSnapshotSave/onSnapshotLoad is used to do Snapshot Operate. 
SnapshotOperation maybe can be a implementation of onSnapshotSave/onSnapshotLoad， it can be delete